sentry/src/main/java/io/sentry/TracingOrigins.java

```diff
@@ -0,0 +1,37 @@
+package io.sentry;
+
+import java.net.URI;
+import java.util.Arrays;
+import java.util.List;
+import org.jetbrains.annotations.NotNull;
+
+/**
+ * Contains a list of origins to which `sentry-trace` header should be sent in HTTP integrations.
+ */
+public final class TracingOrigins {
+  private final @NotNull List<String> origins;
+
+  public TracingOrigins(final @NotNull List<String> origins) {
+    this.origins = origins;
+  }
+
+  public TracingOrigins(final @NotNull String... origins) {
+    this(Arrays.asList(origins));
+  }
+
+  public boolean contain(final @NotNull String url) {
```

Comment:
what's about making it a static method that takes the list and the URL?
otherwise we have to create an instance of TracingOrigins for every check (eg every HTTP call)